lib/Home/data/repository/latest_session/latest_index_session_repository.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:warm_cloud/Home/data/datasource/latest_session/latest_index_session_datasource.dart';
+import 'package:warm_cloud/Home/data/repository/latest_session/latest_session_repository.dart';
+
+class LatestIndexSessionRepository extends LatestSessionRepository {
+  final LatestIndexSessionDatasource latestIndexSessionDatasource;
+
+  LatestIndexSessionRepository(this.latestIndexSessionDatasource);
+
+  @override
+  Future<int> getLatestSessionData() async {
```

Comment:
if the method name suggests it gets data, then why returning index? Please remove index dependency from your project. Index is not reliable for future